modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,29 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
+$ curl https://helloworld-go.default.example.com
+Hello World: Go Sample v1!
+----
+
+== Self signed certificates
+
+If you receive an error relating to a self signed certificate in the certificate chain, you can add the ``--insecure` flag to the curl command to ignore the error.
+
+.Example
+----
+$ curl http://helloworld-go.default.example.com --insecure
+Hello World: Go Sample v1!
+----
+
+== Certificate authority signed certificates
+
+If your {product-title} cluster is configured with a certificate that is signed by a certificate authority (CA), you must specify this with the curl command.
+The path to the certificate can be passed to the curl command by using the ``--cacert` flag.
+
+.Example
+----
+$ curl http://helloworld-go.default.example.com --cacert <certificate-URL>
```

Comment:
Let's change this to `curl http://helloworld-go.default.example.com --cacert <file>` (the same as you see when calling `man curl` and searching for cacert)

Reply:
Actually `curl https://...`